browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -972,8 +970,7 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 	_onTextSelectionMsg: function (textMsg) {
 		L.CanvasTileLayer.prototype._onTextSelectionMsg.call(this, textMsg);
 		// If this is a cellSelection message, user shouldn't be editing a cell. Below check is for ensuring that.
-		if ((this.insertMode === false || this._map._isCursorVisible == false) &&
-		    this._cellCursorXY && this._cellCursorXY.x !== -1) {
+		if ((this.insertMode === false || this._map._isCursorVisible == false) && app.file.calc.cellCursor.visible) {
```

Comment:
Yes, not now unfortunately. Because there is something wrong with the cell cursor for now. It's already in master - cell cursor is visible while editing a cell - i'm not sure if that's expected / wanted. But you are right, let me note this for the future.